pkgs/development/tools/konf/default.nix

```
@@ -0,0 +1,24 @@
+{ lib
+, buildGoModule
+, fetchFromGitHub }:
+
+buildGoModule rec {
+  pname = "konf";
+  version = "0.2.0";
+
+  src = fetchFromGitHub {
+    rev    = "main";
+    owner  = "SimonTheLeg";
+    repo   = "konf-go";
+    sha256 = "sha256-UeuR7lsNG2Y0hdpQA5NXBUlSvYeixyKS73N95z5TZ7k=";
+  };
+
+  vendorSha256 = "sha256-sB3j19HrTtaRqNcooqNy8vBvuzxxyGDa7MOtiGoVgN8=";
+
```

Since #177698 is not yet resolved, I'd recommend setting these flags.
```suggestion

ldflags = [ "-s" "-w" ];

```

---

Good to know, thank you!